src/server/admin/tags/controllers/remove-tag.js

```diff
@@ -0,0 +1,32 @@
+import { removeTagFromService } from '~/src/server/admin/tags/helpers/fetchers.js'
+import { boomify } from '@hapi/boom'
+import Joi from 'joi'
+import { validServiceTags } from '~/src/server/admin/tags/helpers/service-tags.js'
+
+export const removeTagController = {
+  options: {
+    validate: {
+      params: Joi.object({
+        tag: validServiceTags
+      }).unknown(true),
```

Comment:
is this needed? `unknown` is true by default?

Reply:
i thought it was false... but probably doesn't matter